modules/transactions/src/main/java/org/apache/ignite/internal/tx/message/TxStateReplicaRequest.java

```diff
@@ -27,7 +27,7 @@
 /**
  * Transaction state request.
  */
-@Transferable(TxMessageGroup.TX_STATE_REQUEST)
+@Transferable(TxMessageGroup.TX_STATE_REPLICA_REQUEST)
 public interface TxStateReplicaRequest extends ReplicaRequest {
```

Comment:
Better to call it more specific: TxStateCommitPartitionRequest?

Reply:
Agree, I will rename it.